test/Microsoft.TestPlatform.AcceptanceTests/ProcessTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+using Microsoft.TestPlatform.TestUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.TestPlatform.AcceptanceTests;
+
+[TestClass]
+public class ProcessTests : AcceptanceTestBase
```

Comment:
I couldn't really find in which existing type this would fit best so I created this one, feel free to suggest alternative.

Reply:
`ProcessesInteractionTests`?